en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-api-manager.md

```diff
@@ -0,0 +1,320 @@
+
+        ```tab="3.2.0"
+        https://localhost:8243/token
+        https://localhost:8243/revoke
+        ```
+
+        ```tab="4.0.0"
+        https://localhost:9443/oauth2/token
+        https://localhost:9443/oauth2/revoke
+        ```
+
+    - All the data is persisted in databases **from WSO2 API-M 4.0.0 onwards**. Therefore, it is recommended to execute the migration client in the Control Plane profile.
+
+    - From API-M 4.0.0, the kid claim has been removed from the backend JWT.
+  
+        If there are customizations that require validating the kid value of the backend JWT, please customize the JWT generator to include the kid value.
+
+    - From API-M 4.0.0 onwards, Other '_overview' type documents override the API description. If you have created such documents in your lower environments, note that those documents will not be displayed in the document listing page. You can edit those documents the same way you edit the API description.
+
+    - From **WSO2 API_M 4.0.0 onwards**, error responses in API calls have changed from XML to JSON format.
+
+        ??? Attention "If you have developed client applications to handle XML error responses"
+            If you have developed client applications to handle XML error responses you have to change the client applications to handle the JSON responses.
+            
+            For example, for a 404 error response, previously it was as follows.
+    
+            ```xml
+            <am:fault xmlns:am="http://wso2.org/apimanager">
+            <am:code>404</am:code>
+            <am:type>Status report</am:type>
+            <am:message>Not Found</am:message>
+            <am:description>The requested resource is not available.</am:description>
+            </am:fault>
+            ```
+    
+            In API-M 4.0.0 onwards the above resopnse is as follows.
+            
+            ```json
+            {
+            "code":"404",
+            "type":"Status report",
+            "message":"Not Found",
+            "description":"The requested resource is not available."
+            }
+            ```
+
+            For example, for a 404 error response, previously it was as follows.
+    
+            ```xml
+            <am:fault xmlns:am="http://wso2.org/apimanager">
+            <am:code>404</am:code>
+            <am:type>Status report</am:type>
+            <am:message>Not Found</am:message>
+            <am:description>The requested resource is not available.</am:description>
+            </am:fault>
+            ```
+    
+            In API-M 4.0.0 onwards the above resopnse is as follows.
+          
+            ```json
+            {
+            "code":"404",
+            "type":"Status report",
+            "message":"Not Found",
+            "description":"The requested resource is not available."
+            }
+            ```
```

Comment:
This duplicates the information in lines 91-111. Therefore, please delete this section.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6425/commits/50c0db03d6202e339a35013eeebf4e0afdc388ae